<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rectanglify Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="constantAreaCheckBox">
     <property name="toolTip">
      <string>If enabled, rectanglify will scale down the mimimum oriented bounding box as to keep the same area as the original feature</string>
     </property>
     <property name="text">
      <string>Constant area</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="keepRingsCheckBox">
     <property name="toolTip">
      <string>If enabled, rectanglify will try to rectanglify rings in feature geometries</string>
     </property>
     <property name="text">
      <string>Keep rings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="sharedAxesCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>If enabled, the rings will have the same orientation as the outer polygon</string>
     </property>
     <property name="text">
      <string>Rings share axes</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keepRingsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>sharedAxesCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
